docs/UserGuide.md

```diff
@@ -104,6 +117,12 @@ Examples:
 * `eaddv vid/1 eid/1`
 * `eaddv vn/Betsy Crowe en/fundraising`
```

Comment:
we changed to using `n/` only right?

Reply:
i think this one need vn/ and en/ cos the commands involve both events and volunteers

Reply:
ok